README.CentOS6.md

```diff
@@ -56,29 +114,50 @@ To perform this step manually, copy the sample file to the apache configuration
 cp docs/config/apache-site /etc/httpd/conf.d/apache-site.conf
 ```
 
-Ensure that paths and settings in this new file are populated appropriately for your server, e.g. replacing placeholders such as `%LORISROOT%`, `%PROJECTNAME%`, `%LOGDIRECTORY%`. Ensure that the DocumentRoot is pointing to the htdocs/ directory under your LORIS root (usually `/var/www/loris/htdocs`).
+Ensure that paths and settings in this new file are populated appropriately for your server, e.g. replacing placeholders such as `%LORISROOT% --> /var/www/loris`, `%PROJECTNAME% --> loris`, `%LOGDIRECTORY% --> /var/log/httpd/loris-error.log`. Ensure that the DocumentRoot is pointing to `/var/www/loris/htdocs`.
```

Comment:
in what new file? I would change these into bullet points, too.

For instance:

"Ensure the following:
- Paths and settings in [whatever] are populated appropriately for your server (replacing placeholders such as `%LORISROOT%` --> `/var/www/loris`, [etc])
- DocumentRoot is pointing to `/var/www/loris/htdocs`
- The new `smarty/templates_c/` directory under LORIS must be writable by Apache (For instance, by running `chmod 775 /var/www/loris/smarty/templates_c`)"

Whitespace/bullet points are more effective than paragraphs in most studies I know of about how people read documentation.